### ISSUE TITLE
Version on home page is not in maven

### ISSUE THREAD

**Issue by jake-miles** [com-lihaoyi/scalasql#48]
Home page provides this Mill ivy dependency
```
    ivy"com.lihaoyi::scalasql:0.1.13"
```
On Maven Central, the highest version I see published is 0.1.11. This works: 
```
    ivy"com.lihaoyi::scalasql:0.1.11",
```

**Comment by lihaoyi** [comment]
Thanks for the report, seems like the publishing jobs were broken so the artifacts didn't go out. I just pushed a fix and a tag for 0.1.14 so let's see if that works

**Comment by lihaoyi** [comment]
0.1.14 is on maven central
